fastestimator/util/util.py

```
@@ -222,36 +199,27 @@ def teardown() -> None:
-def get_model_parameters(model: Union[tf.keras.Model, torch.nn.Module]) -> Dict[str, int]:
-    if isinstance(model, tf.keras.Model):
-        trainable_params = count_params(model.trainable_weights)
-        non_trainable_params = count_params(model.non_trainable_weights)
-        total_params = trainable_params + non_trainable_params
-    elif isinstance(model, torch.nn.Module):
+def get_model_parameters(model: torch.nn.Module) -> Dict[str, int]:
+    if isinstance(model, torch.nn.Module):
         total_params = sum(p.numel() for p in model.parameters())
         trainable_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
     else:
         raise ValueError("Model not recognized.")
```

Ruff (0.14.3)

Prefer `TypeError` exception for invalid type

(TRY004)